riotfile.py

```diff
@@ -1077,6 +1077,36 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             },
             command="pytest {cmdargs} tests/contrib/aiopg",
         ),
+        Venv(
+            name="aiohttp_contrib",
+            venvs=[
+                Venv(
+                    pys=["3.5","3.6"],
+                    pkgs={
+                        "aiohttp": ["~=2.0", "~=2.1", "~=2.2"],
+                        "aiohttp_jinja": ["~=0.1.2", "~=0.1.3"],
+                        "yarl": "latest",
+                    },
+                ),
+                Venv(
+                    pys=["3.5","3.6", "3.7", "3.8"],
+                    pkgs={
+                        "aiohttp": "~=2.3",
+                        "aiohttp_jinja":"~=0.1.5",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=["3.5","3.6", "3.7", "3.8", "3.9"],
+                    pkgs={
+                        "aiohttp": ["~=3.0", "~=3.1", "~=3.2","~=3.3", "~=3.4", "~=3.5", "~=3.6"],
+                        "aiohttp_jinja": "0.1.5",
```

Comment:
```suggestion
                        "aiohttp_jinja": "~=0.1.5",
```

? the others are that way "~=0.1.5"